libs/@guardian/source-react-components/src/choice-card/ChoiceCard.tsx

```diff
@@ -81,14 +92,50 @@ export const ChoiceCard = ({
 		return !!defaultChecked;
 	};
 
+	const transformProvidedTheme = (
+		providedTheme: typeof choiceCardThemeDefault.choiceCard | undefined,
+	): ChoiceCardTheme => {
+		if (!providedTheme) {
+			return {};
+		}
+		return {
+			textUnSelected: providedTheme.textLabel,
+			textSelected: providedTheme.textChecked,
+			borderUnSelected: providedTheme.border,
+			borderSelected: providedTheme.borderChecked,
+			backgroundSelected: providedTheme.backgroundChecked,
+			...providedTheme,
+		};
+	};
+
+	const getCombinedTheme = (
+		providedTheme: ChoiceCardTheme,
+	): ChoiceCardFullTheme => {
+		if (typeof theme !== 'string' && theme) {
+			return {
+				...choiceCardThemeLight,
+				...providedTheme,
+				...theme,
+			};
+		}
+		if (typeof theme === 'string') {
+			switch (theme) {
+				case 'dark':
+					return choiceCardThemeDark;
+				case 'light':
+					return choiceCardThemeLight;
+			}
+		}
+		return { ...choiceCardThemeLight, ...providedTheme };
+	};
 	// prevent the animation firing if a Choice Card has been checked by default
 	const [userChanged, setUserChanged] = useState(false);
 
 	return (
 		<>
 			<input
 				css={(theme: Theme) => [
-					input(theme.choiceCard),
+					input(getCombinedTheme(transformProvidedTheme(theme.choiceCard))),
```

Comment:
it looks like we always call these functions together - can we not just put the transform inside getCombined theme?